sale_order_digitized_signature/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Order Digitized Signature",
+    "version": "12.0.1.0.0",
+    "author": "Tecnativa, "
+               "Open Source Integrators,"
```

Comment:
@max3903 @mgosai Why You Added `author`, If you migrated then You are a `Contributors`. So, Move in the `Contributors` section.